terraform/terraform-modules/product-copy-completion-checker/lambda/pds-nucleus-product-completion-checker.py

```diff
@@ -191,19 +192,27 @@ def create_harvest_configs_and_trigger_nucleus(list_of_product_labels_to_process
         s3_config_dir = f"s3://{pds_nucleus_config_bucket_name}/dag-data/{random_batch_number}"
         efs_config_dir = f"/mnt/data/dag-data/{random_batch_number}"
 
-        # Create harvest manifets file
+        # Create harvest config file
         harvest_config_xml_content = f"""<?xml version="1.0" encoding="UTF-8"?>
-
-            <harvest nodeName="{pds_node_name}">
-              <files>
-                <manifest>{efs_config_dir + '/' + harvest_manifest_file_path}</manifest>
-              </files>
-              <registry url="{es_url}" index="registry" auth="{es_auth_file}" />
+            <harvest
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="https://github.com/NASA-PDS/harvest/blob/main/src/main/resources/conf/configuration.xsd">
+
+              <!-- Registry configuration -->
+              <!-- UPDATE with your registry information -->
+
+              <registry auth="/etc/es-auth.cfg">file://{efs_config_dir + '/connection.xml'}</registry>
```

Comment:
I thought `file:` URL should be `file:/…` or `file:///…` but if this works, great!